### PR TITLE
Link to the Italian translation of the site added

### PR DESCRIPTION
**Pull request by gabridome**
Parts of the site are not translated yet.

### CODE REVIEW

File: ch4-03-translations.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ contributions under the same license.
 - 🇬🇷 [Chapter I][gr-philosophy], [Chapter II][gr-economics], and [Chapter III][gr-technology] is available in Greek thanks to [nikos].
 - 🇮🇷 [Chapter I, Chapter II, and Chapter III][fa-all] is available in Persian thanks to [Nima].
 - 🇮🇩 [Chapter I][id-philosophy] and [Chapter II][id-economics] is available in Indonesian thanks to [Hazmi Tri Laksono][hazmi]
+- 🇮🇹 Almost [the whole site][it-site], lessons inclueded, is available in Italian thanks to [gabridome]
```
Comment (dergigi) [inline ch4-03-translations.md:43]: typo: inclueded -> included

**Review by dergigi — CHANGES_REQUESTED** [review]
That's awesome, thank you! <3

Please fix the typo if you can and revert the changes outside of `ch4-03-translations.md` (the addition of `| absolute_url`). Again: thank you!

File: ch4-03-translations.md (hunk 1 above)
Comment (gabridome) [reply ch4-03-translations.md:43]: I absolutely agree. I have just to figure out how to correct the PR and I will do.

**Comment by gabridome** [comment]
> That's awesome, thank you! <3
> 
> Please fix the typo if you can and revert the changes outside of `ch4-03-translations.md` (the addition of `| absolute_url`). Again: thank you!

I absolutely agree. I have just to figure out how to correct the PR and I will do.

**Pull request closed without merging**